File: Dragon-WebApi/Models/Campaign.cs
namespace Dragon_WebApi.Models
{
    public class Campaign
    {
        public int Id { get; set; }
        public string CampaignName { get; set; }
        public string CampaignImage { get; set; }
        public DateTime CreationDate { get; set; }

        public Campaign()
        {
            Id = 0;
            CampaignName = "";
            CampaignImage = "";
            CreationDate = DateTime.Now;
        }
    }
}

File: people-among-dragons/src/models/campaign.ts
export class Campaign {
    _campaignId: number;
    _campaignName: string;
    _campaignImage: string;

    constructor(campaignId: number, campaignName: string, campaignImage: string){
        this._campaignId = campaignId;
        this._campaignName = campaignName;
        this._campaignImage = campaignImage;
    }
}